docker/Dockerfile

```diff
@@ -28,7 +30,7 @@ ENV PATH=$CONDA_PREFIX/bin:$PATH
 ENV CONDA_AUTO_UPDATE_CONDA=false
 
 RUN conda install -y ipython
-RUN pip install ninja yacs cython matplotlib
+RUN pip install ninja yacs cython matplotlib opencv-python
 
 # Install PyTorch 1.0 Nightly
 RUN conda install -y pytorch-nightly -c pytorch && conda clean -ya
```

Comment:
Replace with

```
# Install PyTorch 1.0 Nightly and OpenCV
RUN conda install -y pytorch-nightly -c pytorch \
 && conda install -y opencv -c menpo \
 && conda clean -ya
```

and remove your added apt-get install (...)

Reply:
I've tried testing the effect of installing opencv through conda, and it works without my apt-get install xxx, thanks!